util/pingpong.c

```diff
@@ -60,8 +60,12 @@
 #include <rdma/fi_eq.h>
 #include <rdma/fi_errno.h>
 
+#ifndef OFI_MR_BASIC_MAP
+#define OFI_MR_BASIC_MAP (FI_MR_ALLOCATED | FI_MR_PROV_KEY | FI_MR_VIRT_ADDR)
+#endif
+
 #ifndef PP_FIVERSION
-#define PP_FIVERSION FI_VERSION(1, 4)
+#define PP_FIVERSION FI_VERSION(FI_MAJOR_VERSION, FI_MINOR_VERSION)
```

Comment:
We want to keep the hard coded values here, rather than float at the top.  This way we know to update the test prior to updating the version number.  Please set to 1 5